src/results/electric_load.jl

```diff
@@ -1,13 +1,14 @@
 # REopt®, Copyright (c) Alliance for Sustainable Energy, LLC. See also https://github.com/NREL/REopt.jl/blob/master/LICENSE.
 """
 `ElectricLoad` results keys:
-- `load_series_kw` vector of site load in every time step. Does not (currently) include electric load for any new heating or cool techs.
-- `critical_load_series_kw` vector of site critical load in every time step
-- `annual_calculated_kwh` sum of the `load_series_kw`
-- `offgrid_load_met_series_kw` vector of electric load met by generation techs, for off-grid scenarios only
-- `offgrid_load_met_fraction` percentage of total electric load met on an annual basis, for off-grid scenarios only
-- `offgrid_annual_oper_res_required_series_kwh` , total operating reserves required (for load and techs) on an annual basis, for off-grid scenarios only
-- `offgrid_annual_oper_res_provided_series_kwh` , total operating reserves provided on an annual basis, for off-grid scenarios only
+- `load_series_kw` # vector of BAU site load in every time step. Does not include electric load for any new heating or cooling techs.
```

Comment:
@zolanaj  I added these descriptions. If you have a second to review that is welcome!

Reply:
These look good, thank you @adfarth!  I'll merge the PR once tests are confirmed to pass.